test_utilities.py

```diff
@@ -45,8 +47,8 @@ def test_classify_model(self):
             "sTripleScore",
             "class",
         ]
-        id = hashlib.md5(str(self.test_input).encode("utf-8")).hexdigest()[:16]
-        f = features_dict(id, self.test_input)
+        sentence_id = hashlib.md5(str(self.test_input).encode("utf-8")).hexdigest()[:16]
+        f = features_dict(sentence_id, self.test_input)
```

Comment:
`id` is a built-in function. Changing variable name to `sentence_id`
![image](https://user-images.githubusercontent.com/39518771/78592746-2a993300-7863-11ea-9c5d-167adea5511d.png)